Heffner_Daniel/Ruby Assignments/Solar_System/solar_system_spec.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'solar_system'
+RSpec.describe Solar_System do
+
+	describe 'Planet' do
+		before do
+			@tp = Planet.new("test", "test description", 90000000000)
+		end
+
+		it 'has a name' do
+			expect(@tp.name).to eq("test")
+		end
+		it 'has a description' do
+			expect(@tp.description).to eq("test description")
+		end
+		it 'has a population' do
+			expect(@tp.population).to eq(90000000000)
+		end
+	end
+
+	describe 'Solar_System' do
+		before do
+			@ss1 = Solar_System.new
+			@ss2 = Solar_System.new('Alpha Centauri')
+		end
+		it 'has a default name of "Sol"' do
+			expect(@ss1.name).to eq("Sol")
+		end
+		it 'can be initialized with a name' do
+			expect(@ss2.name).to eq("Alpha Centauri")
+		end
+		it 'has a list of planets' do
+			earth = Planet.new("Earth", "Rocky planet", 4000000000000)
+			venus = Planet.new("Venus", "Future home of cloud cities", 0)
+			@ss1.add_planet(earth)
+			@ss1.add_planet(venus)
+			expect(@ss1.planet_list).to eq([earth, venus])
```

Comment:
I'd be a little careful due to order on this one. Highly dependent on which order you add rather than just checking if they are there.